### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
- Create a method called current_user to make current user data available to all controllers.

- Create forms to perform the following functions:
1. Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
2. Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).

- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.first
```
Comment (gemmen29) [inline app/controllers/application_controller.rb:3]: - [ ] Kindly allow users to add likes to posts as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Allow%20Users%20to%20add%20likes%20to%20Posts.).

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
```
Comment (gemmen29) [inline app/controllers/application_controller.rb:2]: - [ ] Kindly allow user to create comment on behalf on the `current_user` as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Create%20a%20comment%20on%20behalf%20of%20the%20current_user%20(use%20the%20method%20that%20you%20created%20in%20your%20ApplicationController ).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @maenkhraisha, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (maen-al-khraisha) [reply app/controllers/application_controller.rb:3]: thank you, I add it

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (maen-al-khraisha) [reply app/controllers/application_controller.rb:2]: thank you, I add it

**Review by smunozmo — APPROVED** [review]
Hi @maenkhraisha, 👋 

## Highlights 👏 

- Excellent job making the suggested changes! 😎 
- Forms are set up correctly, great! 🎉 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**